css/properties/height.json

```diff
@@ -0,0 +1,261 @@
+{
+  "css": {
+    "properties": {
+      "height": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/height",
+          "support": {
+            "webview_android": {
+              "version_added": "1"
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "46"
```

Comment:
This is '1' (was true in the original table, but height is existing since the beginning).